src/main/java/Duke.java

```diff
@@ -1,10 +1,59 @@
+import java.util.Scanner;  // Import the Scanner class
```

Comment:
I see many comments in the code that made be redundant since the function names or variables names themselves are quite self-explanatory. So consider removing redundant comments. [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#guideline-comment-minimally-but-sufficiently)